### ISSUE TITLE
Search Page: Further enhance design

### ISSUE THREAD

**Issue by mariobehling** [fossasia/open-event-frontend#6017]
Please enhance the design of the search page as shown in the screenshot below. Change the following:
* Move search bar to middle top menu bar
* Take out heading "Events" above search results
* Take out search box from left hand side menu bar
* rename Location Type to Setting

compare https://eventyay.com/explore
![Screenshot from 2020-12-13 12-25-56](https://user-images.githubusercontent.com/1583873/102010692-ff46dc00-3d3f-11eb-9293-0b6d3e5760eb.png)

**Comment by nlok5923** [comment]
Working on it sir.

**Comment by nlok5923** [comment]
@mariobehling sir do I need to move the search box of side bar into the middle of navbar. Or it should be above event cards.

**Comment by Afzal-Ind** [comment]
Need collaboration! I want to work on this.

**Comment by toHarsh** [comment]
I can work on this, also with @nlok5923 if needed!

**Comment by Afzal-Ind** [comment]
@toHarsh @nlok5923 Our next jo is to **Move search bar to middle top menu bar** and **Take out search box from left hand side menu bar**. You can reply here if you want to try else I will try. Plz see this screenshot. 
![c4](https://user-images.githubusercontent.com/11625672/102060567-5c15c580-3e18-11eb-8a45-eaf3ffb14d38.JPG)

**Comment by Harshit564** [comment]
@Afzal-Ind I am trying the search bar part

**Comment by Afzal-Ind** [comment]
> I am trying the search bar part

Nice, you can also try. But remember to add my changes on your end and then make changes the left ones as per the suggestion. Thanks.

**Comment by Harshit564** [comment]
> > I am trying the search bar part
> 
> Nice, you can also try. But remember to add my changes on your end and then make changes the left ones as per the suggestion. Thanks.

Okay will do that

**Comment by daretobedifferent18** [comment]
@mariobehling  This issues can be solved after #6007 in a better way

**Comment by toHarsh** [comment]
> 
> 
> @toHarsh @nlok5923 Our next jo is to **Move search bar to middle top menu bar** and **Take out search box from left hand side menu bar**. You can reply here if you want to try else I will try. Plz see this screenshot.
> ![c4](https://user-images.githubusercontent.com/11625672/102060567-5c15c580-3e18-11eb-8a45-eaf3ffb14d38.JPG)

sure...i'll do it!

**Comment by toHarsh** [comment]
i removed the search bar from left hand side menu bar.

**Comment by Afzal-Ind** [comment]
> sure...i'll do it!


Sorry it's been already solved. You are late to reply. Look for another issues now.

**Comment by toHarsh** [comment]
> 
> 
> > sure...i'll do it!
> 
> Sorry it's been already solved. You are late to reply. Look for another issue now.

okay no worries!

**Comment by Manav1** [comment]
Can I work on the issue?
